lib/new-admin/config/accounts.js

```diff
@@ -38,8 +38,8 @@ const ALL_ACCOUNTS = [
   { code: 'mock-id-verify', display: 'Mock ID verifier', class: ID_VERIFIER, dev: true },
   { code: 'twilio', display: 'Twilio', class: SMS },
   { code: 'mailgun', display: 'Mailgun', class: EMAIL },
-  { code: 'all-zero-conf', display: 'Always 0-conf', class: ZERO_CONF, cryptos: [BTC, ZEC, LTC, DASH, BCH] },
-  { code: 'no-zero-conf', display: 'Always 1-conf', class: ZERO_CONF, cryptos: [ETH] },
+  { code: 'all-zero-conf', display: 'None', class: ZERO_CONF, cryptos: [BTC, ZEC, LTC, DASH, BCH] },
+  { code: 'no-zero-conf', display: 'None', class: ZERO_CONF, cryptos: [ETH] },
```

Comment:
In this case, maybe we should collapse this in the code, as well to ``no-confidence-checking``, ``blockcypher`` and ``mock-confidence-checking``.